cmd/bundle.go

```diff
@@ -78,6 +84,11 @@ func init() {
 	bundleBuildCmd.Flags().StringP("build-directory", "b", ".", "Path to a directory containing a massdriver.yaml file.")
 	bundleCmd.AddCommand(bundlePublishCmd)
 	bundlePublishCmd.Flags().StringP("build-directory", "b", ".", "Path to a directory containing a massdriver.yaml file.")
+	bundleNewCmd.Flags().StringP("name", "n", "", "Name of the new bundle")
+	bundleNewCmd.Flags().StringP("description", "d", "", "Description of the new bundle")
+	bundleNewCmd.Flags().StringP("template-type", "t", "", "Name of the bundle template to use")
+	bundleNewCmd.Flags().StringSliceP("connections", "c", []string{}, "Connections and names to add to the bundle - example: massdriver/vpc=network")
```

Comment:
This will need to be the opposite direction so that a bundle can have multiple of the same type connection... `network=massdriver/aws-vpc` ...